lib/bamboo/email.ex

```diff
@@ -117,7 +117,12 @@ defmodule Bamboo.Email do
 
   for function_name <- @address_functions do
     @doc """
-    Sets the #{function_name} on the email
+    Sets the `#{function_name}` on the email
+
+    ```
+    new_email
+    |> #{function_name}(["sally@example.com", "james@example.com"])
```

Comment:
Added a PR to change those examples: https://github.com/thoughtbot/bamboo/pull/165